src/sentry/deletions/base.py

```diff
@@ -125,7 +145,6 @@ def delete_bulk(self, instance_list) -> bool:
         self.mark_deletion_in_progress(instance_list)
 
         child_relations = self.get_child_relations_bulk(instance_list)
-        child_relations = self.extend_relations_bulk(child_relations, instance_list)
```

Comment:
Is this not needed anymore?

Reply:
Nope. No deletion tasks have implemented this method.